lib/cs_guide/resources/venue.ex

```diff
@@ -35,7 +43,20 @@ defmodule CsGuide.Resources.Venue do
   @doc false
   def changeset(venue, attrs \\ %{}) do
     venue
-    |> cast(attrs, [:venue_name, :postcode, :phone_number, :cs_score])
+    |> cast(attrs, [
+      :venue_name,
+      :postcode,
+      :phone_number,
+      :cs_score,
+      :description,
+      :num_cocktails,
+      :website,
+      :address_line_1,
```

Comment:
again, the `_1` in `address_line_1` is only relevant in  a world where we want to capture multiple lines of address in _separate_ fields, which we don't. I would _prefer_ this to be tidy.